app/Jobs/GitHub/Populate/PopulateOrganizationsJob.php

```
@@ -2,15 +2,42 @@
+use App\Jobs\GitHub\Sync\SyncOrganizationJob;
 use App\Services\GitHub\Contracts\GitHubClientContract;
-use App\Services\GitHub\Contracts\GitHubRepositoriesClientFactoryContract;
-use App\Transformers\GitHub\RepositoryTransformer;
+use App\Services\GitHub\Contracts\GitHubSubClientContract;
 
-class PopulateRepositoriesJob extends AbstractPopulateJob
+class PopulateOrganizationsJob extends AbstractPopulateJob
 {
     public function __construct(
-        private int $organizationId,
         private bool $force = false
```

PHPMD (2.15.0)

The method __construct has a boolean flag argument $force, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)